src/runtime/ltypes/ltypes.py

```diff
@@ -92,6 +92,9 @@ def overload(f):
     return overloaded_f
 
 
+def interface(f):
+    raise Exception("Unexpected to be called by CPython")
```

Comment:
I think this will raise the exception already when you type `@interface`. But rather, it should raise this only if you actually call it, not just declare it.